doc/source/templates/04_finetuning_llms_with_deepspeed/cluster_env.yaml

```diff
@@ -1,31 +1,33 @@
 # See https://hub.docker.com/r/anyscale/ray for full list of
 # available Ray, Python, and CUDA versions.
-base_image: anyscale/ray:2.5.1-py39-cu116
+base_image: {{ env["RAY_IMAGE_ML_NIGHTLY_GPU"] | default("anyscale/ray:2.6.1-py39-cu117") }}
```

Comment:
why use nightly? why not use a released one?

also why not use cu118?

Reply:
to always test this with the latest on CI?